tests/torch/data/reference_graphs/quantized/synthetic_model/MultiOutputSameTensorModel.dot

```diff
@@ -1,13 +1,16 @@
 strict digraph  {
 "0 /nncf_model_input_0" [id=0, type=nncf_model_input];
-"1 SymmetricQuantizer/symmetric_quantize_0" [id=1, type=symmetric_quantize];
-"2 MultiOutputSameTensorModel/__mul___0" [id=2, type=__mul__];
-"3 /nncf_model_output_0" [id=3, type=nncf_model_output];
-"4 /nncf_model_output_1" [id=4, type=nncf_model_output];
-"5 /nncf_model_output_2" [id=5, type=nncf_model_output];
-"0 /nncf_model_input_0" -> "1 SymmetricQuantizer/symmetric_quantize_0";
-"1 SymmetricQuantizer/symmetric_quantize_0" -> "2 MultiOutputSameTensorModel/__mul___0";
-"1 SymmetricQuantizer/symmetric_quantize_0" -> "3 /nncf_model_output_0";
-"1 SymmetricQuantizer/symmetric_quantize_0" -> "5 /nncf_model_output_2";
-"2 MultiOutputSameTensorModel/__mul___0" -> "4 /nncf_model_output_1";
+"1 MultiOutputSameTensorModel/SymmetricQuantizer/symmetric_quantize_0" [id=1, type=symmetric_quantize];
+"2 MultiOutputSameTensorModel/SymmetricQuantizer/symmetric_quantize_1" [id=2, type=symmetric_quantize];
+"3 MultiOutputSameTensorModel/__mul___0" [id=3, type=__mul__];
+"4 /nncf_model_output_0" [id=4, type=nncf_model_output];
+"5 /nncf_model_output_1" [id=5, type=nncf_model_output];
+"6 /nncf_model_output_2" [id=6, type=nncf_model_output];
+"0 /nncf_model_input_0" -> "1 MultiOutputSameTensorModel/SymmetricQuantizer/symmetric_quantize_0";
+"0 /nncf_model_input_0" -> "2 MultiOutputSameTensorModel/SymmetricQuantizer/symmetric_quantize_1";
+"0 /nncf_model_input_0" -> "4 /nncf_model_output_0";
+"0 /nncf_model_input_0" -> "6 /nncf_model_output_2";
+"1 MultiOutputSameTensorModel/SymmetricQuantizer/symmetric_quantize_0" -> "3 MultiOutputSameTensorModel/__mul___0";
+"2 MultiOutputSameTensorModel/SymmetricQuantizer/symmetric_quantize_1" -> "3 MultiOutputSameTensorModel/__mul___0";
+"3 MultiOutputSameTensorModel/__mul___0" -> "5 /nncf_model_output_1";
 }
```

Comment:
Since the fix introduces new behaviour - guaranteeing that outputs are not quantized by default - the new graph is correct. That this leads to one and the same tensor being quantized twice with the same parameters is, unfortunately, a limitation of the hooking approach in NNCF PT in this exact scenario. We can either quantize the output of the `nncf_model_input_0` node, which will lead to outputs being quantized too, or we can quantize the two inputs of the `__mul__0` node, which will lead to duplication of effort, but will get us to the desired situation where the outputs are not quantized.